modules/nw-ptp-configuring-linuxptp-services-as-ordinary-clock.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple_networks/configuring-ptp.adoc
+
+[id="configuring-linuxptp-services-as-ordinary-clock_{context}"]
+= Configuring Linuxptp services as Ordinary Clock
+
+The PTP Operator adds the `PtpConfig.ptp.openshift.io` custom resource definition (CRD) to {product-title}.
+You can configure the linuxptp services (ptp4l, phc2sys) by creating a `PtpConfig` custom resource (CR) object.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+* Install the PTP Operator.
+
+.Procedure
+
+. Create the following `PtpConfig` CR, and then save the YAML in the `<name>-ptp-config.yaml` file. Replace `<name>` with the name for this configuration.
++
+[source,yaml]
+----
+apiVersion: ptp.openshift.io/v1
+kind: PtpConfig
+metadata:
+  name: <name> <1>
```

Comment:
ordinary-clock-ptp-config.yaml

Reply:
Do I include the file format for this field or not? In the other file, you didn't include it. Isn't this the name of the CR?

Reply:
no, just `ordinary-clock-ptp-config`

Reply:
Apologies! typo on my end